docs/query-classification-and-display.md

```diff
@@ -0,0 +1,125 @@
+data flow could make an alert switch from AlwayNull to MaybeNull (or
+vice versa). As a result we attribute both a fix and an introduction to
+the developer that changed the data flow. For this particular example
+the funny attribution results are more a nuisance than a real problem;
+the overall alert count remains unchanged. However, for the duplicate
+and similar code queries the effects can be much more severe, as they
+come in versions for "duplicate file" and "duplicate function" among
+many others, where "duplicate function" only triggers if "duplicate
+file" didn't. As a result adding some code to a duplicate file might
+result in a "fix" of a "duplicate file" alert and an introduction of
+many "duplicate function" alerts. This would be highly unfair.
+Currently, on the duplicate and similar code queries exhibit this
+"exchanging one for many" alerts when trying to attribute their results.
+Therefore we currently exclude all duplicate code related alerts from
+attribution.
+
+The following queries are excluded from attribution:
+
+  - Metric violations, i.e. the ones with metadata properties like ` 
+    @(error|warning|recommendation)-(to|from)  `
+  - Queries with tag `  non-attributable `
+
+`   `
+
+<div>
+
+<div>
+
+This check is applied when the results of a single attribution are
+loaded into the datastore. This means that any change to this behaviour
+will only take effect on newly attributed revisions and the historical
+data is unchanged.
+
+</div>
+
+</div>
+
+ 
+
+Query severity and precision
+
+We currently classify queries in on two axes, with some additional tags.
+Those axes are severity and precision, and are defined using the
+query-metadata properties `@problem.severity` and `@precision`.
+
+For severity, we have the following categories:
+
+  - Error
+  - Warning
+  - Recommendation
+
+These categories may change in the future.
+
+For precision, we have the following categories:
+
+  - Very-high
+  - High
+  - Medium
+  - Low
+
+As [usual](https://en.wikipedia.org/wiki/Precision_and_recall),
+precision is defined as the percentage of query results that are true
+positives, i.e., precision = number of true positives / (number of true
+positives + number of false positives). There is no hard-and-fast rule
+for which precision ranges correspond to which categories.
+
+We expect these categories to remain unchanged for the forseeable
+future.
+
+### A note on precision
+
+Intuitively, precision measures how well the query does at finding the
+results it is supposed to find, i.e., how well it implements its
+(informal, unwritten) rule. So how precise a query is depends very much
+on what we consider that rule to be. We generally try to sharpen our
+rules to focus on results that a developer might actually be interested
+in.
+
+  
+
+## Which queries to run and display on LGTM
+
+The following queries are run:
+
+<div class="table-wrap">
+
+| Precision:     | V. high     | High    | Medium  | Low |
```

Comment:
"V. high" could be written out as "Very-high" since the table is not that wide.
Same applies for the other table as well.